### DISCUSSION TITLE
模型训练出现问题

### DISCUSSION THREAD

**Discussion by ReneeQRDu** [Congyuwang/AlphaNetV3#15] [Q&A] [answered]
[Hi我在按照这里的指引](https://github.com/Congyuwang/AlphaNetV3)训练模型的时候，出现了一个问题，如下：
![1635918532(1)](https://user-images.githubusercontent.com/74351509/140014906-be6fd4ed-d72d-4be0-bd5d-214ab616bd85.png)
想问问这是怎么回事
电脑已经安装了numba

**Comment by ReneeQRDu** [comment]
这个错误在我运行自带的测试文件的时候也遇到了

**Comment by Congyuwang** [comment, marked as answer]
在我的平台上正常工作。请尝试debug，欢迎pull request
